src/Native/jitinterface/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ set(NATIVE_SOURCES
     corinfoexception.cpp
 )
 
-if(WIN32)
+if(WIN32 AND NOT CLR_CMAKE_PLATFORM_ARCH_I386)
```

Comment:
Nit: `and CLR_CMAKE_PLATFORM_ARCH_AMD64` maybe better

Reply:
Changed; thought aarch64 may have same sources, but I guess that architecture isn't yet supported. :)